app/views/devise/sessions/new.html.slim

```diff
@@ -1,4 +1,5 @@
 h2 Log in
+= link_to "Sign in with Google", user_google_oauth2_omniauth_authorize_path
```

Comment:
@marderer let's move all plain text like `Sign in with Google` to `en.yml` locales